rails/benchmarks/activerecord_complex.rb

```diff
@@ -0,0 +1,103 @@
+require 'bundler/setup'
+
+require 'benchmark/ips'
+require 'json'
+
+require 'rails'
+require 'action_controller/railtie'
+require 'active_record'
+require 'sqlite3'
+require 'ffaker'
+
+TIME = (ENV['BENCHMARK_TIME'] || 5).to_i
```

Comment:
I think 5 or 10 is fine. I just noticed that another script was set to 10, so I figured it'll be good to keep all defaults the same :smile: